opt/jvmcommon.sh

```diff
@@ -55,10 +55,10 @@ if echo "${JAVA_OPTS:-}" | grep -q "\-Xmx"; then
 else
   default_java_opts="${default_java_mem_opts} -Dfile.encoding=UTF-8"
   export JAVA_OPTS="${default_java_opts} ${JAVA_OPTS:-}"
-  if echo "${DYNO}" | grep -vq '^run\..*$'; then
+  if echo "${DYNO}" | grep -vq '^one-off-*$'; then
```

Comment:
You must not replace both `..`, only the first one is for the label `web.1` for instance, the second goes with the `*` it's the regexp part `.*$`

Reply:
Silly me, thanks